assets/variant-list.css

```diff
@@ -0,0 +1,370 @@
+    visibility: hidden;
+  }
+
+  .variant-item__quantity:hover variant-list-remove-button,
+  .variant-item:has(.quantity__input:focus-visible) variant-list-remove-button {
+    visibility: visible;
+  }
+
+  .variant-item__image {
+    max-width: 100%;
+  }
+
+  .variant-item__contain {
+    display: flex;
+  }
+
+  .variant-item__discounted-prices {
+    justify-content: flex-end;
+  }
+}
+
+.variant-item__details {
+  font-size: 1.6rem;
+  line-height: calc(1 + 0.4 / var(--font-body-scale));
+}
+
+.variant-item__details>* {
+  margin: 0;
+  max-width: 30rem;
+}
+
+
+.variant-item__info {
+  position: relative;
+}
+
+.variant-item__name {
+  display: block;
+}
+
+.variant-item__discounted-prices {
+  margin-top: 0;
+  margin-bottom: 0;
+  display: flex;
+  align-items: center;
+}
+
+.variant-item__discounted-prices dd {
+  margin: 0;
+}
+
+.variant-item__discounted-prices dd:first-of-type {
+  margin-right: 1rem;
+}
+
+.variant-item__discounted-prices .variant-item__old-price {
+  font-size: 1.4rem;
+}
+
+.variant-item__old-price {
+  opacity: 0.7;
+}
+
+.variant-item__final-price {
+  font-weight: 400;
+}
+
+.product-option {
+  font-size: 1.4rem;
+  word-break: break-word;
+  line-height: calc(1 + 0.5 / var(--font-body-scale));
+}
+
+.variant-item__quantity-info {
+  display: flex;
+  align-items: center;
+  margin: 0 1.5rem 0 0;
+  min-width: 1.5rem;
+  min-height: 1.5rem;
+}
+
+.variant-item__sold-out {
+  opacity: 0.7;
+}
+
+.variant-item variant-list-remove-button {
+  display: flex;
+  margin-left: 1rem;
+  align-self: center;
+}
+
+@media screen and (min-width: 750px) and (max-width: 989px) {
+  .variant-item variant-list-remove-button {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+}
+
+variant-list-remove-button .button {
+  min-width: calc(1.5rem / var(--font-body-scale));
+  min-height: 1.5rem;
+  padding: 0;
+  margin: 0 0.1rem 0.1rem 0;
+}
+
+
+variant-list-remove-button .button:not([disabled]):hover {
+  color: rgb(var(--color-foreground));
+}
+
+@media screen and (min-width: 750px) {
+  variant-list-remove-button .button {
+    min-width: 1.5rem;
+    min-height: 1.5rem;
+  }
+}
+
+variant-list-remove-button .icon-remove {
+  height: 1.5rem;
+  width: 1.5rem;
+}
+
+.variant-item .loading-overlay {
+  top: 0;
+  left: auto;
+  right: 0;
+  bottom: 0;
+  padding: 0;
+}
+
+
+@media screen and (min-width: 750px) {
+  .variant-item .loading-overlay {
+    padding-top: 2.5rem;
+    bottom: auto;
+  }
+}
+
+.variant-item .loading-overlay:not(.hidden)~* {
+  visibility: hidden;
+}
+
+.variant-item__error {
+  display: flex;
+  align-items: flex-start;
+  margin-top: 1rem;
+}
+
+.variant-item__error-text {
+  font-size: 1.2rem;
+  order: 1;
+}
+
+.variant-item__error-text+svg {
+  flex-shrink: 0;
+  width: 1.2rem;
+  margin-right: 0.7rem;
+}
+
+.variant-item__error-text:empty+svg {
+  display: none;
+}
+
+.product-option+.product-option {
+  margin-top: 0.4rem;
+}
+
+.product-option * {
+  display: inline;
+  margin: 0;
+}
+
```

Comment:
Ugh.. is all this type of stuff duplicated from the cart styles? I see what you mean a refactor could be in order, but can we find a middle ground for now? Like maybe consider utilizing that stylesheet for this section too, by conditionally loading it on pdp and using the existing `.cart-item ...` and `.product-option` classnames in the variant list markup? 

We can still keep the variant specific classes if we need specific styles for that, but including the existing classes would probably prevent a lot dupes. `<tr class="variant-item cart-item">`

Reply:
> The styles are slightly different and to avoid increasing the scope of testing, it's something that will be a fast follow up and not something in this PR

I'll leave it to you, but this seems like the kind of thing that would be get out of the way now. It would highlight the differences between the two cases that we should test, make every PR built on this quicker and easier to review, and perhaps most importantly, would be safer than having to go back later and risk introducing regressions.